service/service.go

```diff
@@ -62,6 +62,11 @@ func New(config Config) (*Service, error) {
 	{
 		healthzConfig := healthz.DefaultConfig()
 
+		if config.Flag.Service.CheckAPI == "true" {
```

Comment:
```suggestion
		if config.Flag.Service.CheckAPI == strings.ToLower("true") {
```
Having case insensitivity here may save some time in the future when being unsure if it should be `TRUE` or `true`.